sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -829,6 +829,50 @@ val GPU_COREDUMP_PIPE_PATTERN = conf("spark.rapids.gpu.coreDump.pipePattern")
     .bytesConf(ByteUnit.BYTE)
     .createWithDefault(8 * 1024 * 1024)
 
+  // ASYNC PROFILER (FOR FLAME GRAPH)
+
+  val ASYNC_PROFILER_PATH_PREFIX = conf("spark.rapids.flameGraph.pathPrefix")
+    .doc("Enables collecting flame graph (with async profiler) and specifies " +
+      "a file prefix to use when writing the JFR file by async-profiler. " +
+      "The async-profiler will write a flame graph file for each stage. " +
+      "It is strongly recommended to set 'spark.scheduler.mode' to 'FIFO' " +
+      "so that there is a clean boundary between stages, " +
+      "and then we can better understand each stage.")
+    .stringConf
+    .createOptional
+
+  val ASYNC_PROFILER_EXECUTORS = conf("spark.rapids.flameGraph.executors")
```

Comment:
nit. we can make the name consistent. e.g., `spark.rapids.asyncProfiler.executors` , `spark.rapids.asyncProfiler.options`, `spark.rapids.asyncProfiler.compression` and `spark.rapids.asyncProfiler.stageEpochInterval`

Reply:
the config naming convention was also a result of Jihoon's advise